src/index.js

```diff
@@ -400,6 +400,11 @@ const Trigger = createReactClass({
     return childPros[event] || props[event];
   },
 
+  isClickedOnScrollBar(clientX) {
+    return clientX >= document.documentElement.offsetWidth ||
+      event.clientY >= document.documentElement.clientHeight;
```

Comment:
You should link your modified rc-trigger in rc-tooltip, and test if your modification works before PR, thanks.